pkg/destroy/gcp/gcp.go

```diff
@@ -130,8 +130,12 @@ func (o *ClusterUninstaller) Run() (*types.ClusterQuota, error) {
 		return nil, errors.Wrap(err, "failed to create resourcemanager service")
 	}
 
-	err = wait.PollImmediateInfinite(
+	pollCtx, cancel := context.WithTimeout(context.Background(), time.Minute*60)
```

Comment:
We should probably make this timeout a constant somewhere in the destroy package so other clouds also have access to it in case we implement something similar for them.

Reply:
If we decide to do this for all clouds should we add them to this PR as separate commits or make individual PRs? @patrickdillon @JoelSpeed

Reply:
Having a destroy timeout const makes sense, keeping scope to one cloud at a time also makes sense. Where shall we put the const so we can use it later?

Reply:
We can probably add it to destroy as a single file. The const value(s) could be added there.

Reply:
Following the existing pattern in this file, could you add this new timeout here? https://github.com/openshift/installer/pull/7441/files#diff-3ec5133e651fafaef3079f9885802bcf90656974de0f06fe8be654af182b288eR32

Also, I checked, aws and azure. They don't seem to have any constants defined in their destroy code. If we were going to add a common location for defining constants in the /pkg/destroy/ path that could be used for multiple platforms, I don't see any other constants from other platforms that can utilize this.